service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -47,9 +50,23 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
                 "JOIN \n" +
                 "   itembank.testformitem FI \n" +
                 "   ON FI._fk_item = I._key \n" +
-                "JOIN " +
+                "JOIN \n" +
                 "   itembank.tblsetofadminsubjects segments \n" +
                 "   ON segments._key = A._fk_adminsubject \n" +
+                "JOIN \n" +
+                "   itembank.tblclient C \n" +
```

Comment:
I know you're following the pattern from before but I think with the size of these queries that it may be beneficial to name these aliases rather than using single letters.  It still can be short but a work my help.

- A -> adminItems
- I -> item
- FI -> formItem
- C -> client
- B -> bank
- S -> stimulus